src/main/java/org/apache/brooklyn/cloudfoundry/entity/VanillaCloudFoundryApplication.java

```diff
@@ -43,9 +43,10 @@
 @ImplementedBy(VanillaCloudFoundryApplicationImpl.class)
 public interface VanillaCloudFoundryApplication extends Entity, Startable, DriverDependentEntity {
 
-    @SetFromFlag("name")
-    ConfigKey<String> APPLICATION_NAME = ConfigKeys.newStringConfigKey(
-            "cloudFoundry.application.name", "Name of the application");
+    @SetFromFlag("name-app")
```

Comment:
I have fix the flag to `nameApp` while we decide what to do about the `name` flag